include/stp/c_interface.h

```diff
@@ -1136,6 +1136,47 @@ DLL_PUBLIC int getExprID(Expr ex);
 DLL_PUBLIC int vc_parseMemExpr(VC vc, const char* s, Expr* outQuery,
                                Expr* outAsserts);
 
+//! \brief Sets underlying SAT solver to minisat
+//!
+DLL_PUBLIC unsigned char vc_setMinisat(VC vc);
```

Comment:
Can you rename these to `vc_useMinisat` (and so on), which reads a little bit better?

Reply:
Yep!